worker/worker.go

```diff
@@ -29,6 +31,42 @@ type Worker struct {
 	QueueId        string
 }
 
+type TaskState uint8
+
+const (
+	TaskPending TaskState = iota
+	TaskRunning
+	TaskFinished
+	TaskFailed
+)
+
+//This struct represents a task, the executable piece of the system.
+type Task struct {
+	// This array wraps the task's commands. Each positions of the array
+	// is a unix command.
+	Commands       []string
+	// This field stands for the the time interval in which
```

Comment:
rename to something like `period (in seconds) between report status from the worker to server

Reply:
done